designer/wizard-xul/src/main/java/org/pentaho/reporting/engine/classic/wizard/ui/xul/steps/LookAndFeelStep.java

```diff
@@ -582,7 +583,10 @@ public Integer getSelectedTemplate() {
     return selectedTemplate;
   }
 
-  public void setSelectedTemplate( final Integer selectedTemplate ) {
+  public void setSelectedTemplate( Integer selectedTemplate ) {
+    if ( selectedTemplate == -1 ) {
```

Comment:
If the problem was the image control not being initially in sync, calling `imageBinding.fireSourceChanged()`, above, should fix the issue.
So why do you need this here?

Reply:
Done